test/scripts/e2e_subs/serial/rest-proof-endpoint.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+# TIMEOUT=600
```

Comment:
do we really need a timeout of 10m for this one ?

Reply:
Thanks, forgot about this. A long timeout is no longer required.